src/server/game/AuctionHouse/AuctionHouseMgr.cpp

```diff
@@ -892,6 +896,21 @@ bool AuctionEntry::LoadFromDB(Field* fields)
         TC_LOG_ERROR("misc", "Auction %u has not a existing item : %u", Id, itemGUIDLow);
         return false;
     }
+
+    // get list of bidders
+    PreparedStatement* stmt = CharacterDatabase.GetPreparedStatement(CHAR_SEL_AUCTION_BIDDERS);
+    stmt->setUInt32(0, Id);
+
+    if (PreparedQueryResult result = CharacterDatabase.Query(stmt))
```

Comment:
All auctions are loaded on startup with one query in `AuctionHouseMgr::LoadAuctions`, just run another query (for the entire table) there and parse it before the loop calling `AuctionEntry::LoadFromDB`